aquadrone_pneumatics/nodes/pneumatics_node.py

```diff
@@ -0,0 +1,4 @@
+from aquadrone_pneumatics import Pneumatics
+
+if __name__ == "main":
+    pneumatics_contoller = Pneumatics()
```

Comment:
Add a line with `pneumatics_controller.run()`. The corresponding function there can just call `rospy.spin()`. This will basically tell ROS to prevent this node from exiting (because how it is set up right now it will initialize, then the code will immediately end right after).

Reply:
c9b0735